src/main/resources/config/application.yaml

```diff
@@ -0,0 +1,5 @@
+spring:
+  application:
+    name: case-import-server
+
+target-directory-name: Automatic_cases_import
```

Comment:
Why do you need this ?

Reply:
Ah yes, that's the name of the targeted directory.
So this directory name will be configured in the deployment files ?

Reply:
yes it can be overwritten